ipaclient/install/ipa_epn.py

```diff
@@ -238,12 +238,35 @@ def add_options(cls, parser):
 
     def validate_options(self):
         super(EPN, self).validate_options(needs_root=True)
-        if self.options.to_nbdays:
+        if self.options.to_nbdays is not None:
+            try:
+                int(self.options.to_nbdays)
+                assert float(self.options.to_nbdays) == \
+                    int(self.options.to_nbdays)
+            except Exception as e:
+                self.option_parser.error(
+                    "--to-nbdays must be an integer. {error}".format(error=e)
+                )
             self.options.dry_run = True
-        if self.options.from_nbdays and not self.options.to_nbdays:
+        if self.options.from_nbdays is not None:
+            try:
+                int(self.options.from_nbdays)
```

Comment:
Is it possible to provide negative values, for instance to check the passwords that expired last week? If no, then you should also check that int(self.options.from_nbdays)>=0

Reply:
I'd rather avoid this for now, so I've added the check.